components/Collapsible.tsx

```
@@ -25,7 +24,7 @@ export function Collapsible({ children, title }: PropsWithChildren & { title: st
           style={{ transform: [{ rotate: isOpen ? '90deg' : '0deg' }] }}
         />
 
-        <ThemedText type="defaultSemiBold">{title}</ThemedText>
+        <Text type="defaultSemiBold">{title}</Text>
```

ESLint

[error] 'Text' is not defined.

(react/jsx-no-undef)